client/src/main/scala/drt/client/components/FlightTable.scala

```diff
@@ -50,69 +45,115 @@ object FlightTable {
                    viewEnd: SDateLike,
                    showFlagger: Boolean,
                    paxFeedSourceOrder: List[FeedSource],
-                   filterFlightNumber: String) extends UseValueEq
+                   filterFlightNumber: String,
```

Comment:
How about we add the class containing these 6 properties (`FlightHighlight`) rather than them individually. It'll mean 5 fewer parameters in a few places